README.md

```diff
@@ -74,26 +74,49 @@ We use the [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/late
 to store secrets for this project.  An AWS best practice is to create secrets
 with a unique ID to prevent conflicts when multiple instances of this project
 is deployed to the same AWS account.  Our naming convention is
-`<cfn stack name>/<environment id>/<secret name>`.  An example is `myapp/dev/MySecret`
+`<cfn stack name>/<environment id>/<secret name>`.  The template in this repo' uses
+the secret name, `ecs`, so an example Secrets Manager name is `myapp/dev/ecs`.
+
+
+## Deployment from GitHub to AWS
+
+To allow the GitHub action in your repository to deploy infrastructure to AWS, submit a
+PR like [this](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/771/files),
+customizing the `StackName`, `Repositories` (name), and the `Account`(s) to 
+in which the infrastructure will be deployed.
+
+Once the PR is merged, an IAM role will be created in each AWS account specified in the PR.  
+The ARN for the role will go into the `role-to-assume` field of `aws-deploy.yml`.  
+NOTE:  To separately deploy into different AWS accounts (say, `dev`, `test`, and `prod`), 
+create separate GitHub branches and customize `aws-deploy.yml` in each branch with the ARN 
+of the role for the target AWS account.
+
+
+## VPC CIDR
+
+Select a unique IP address range for the VPC (CIDR) following the instructions 
+[here](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2850586648/Setup+AWS+VPC).
+Enter the range as the `VPC_CIDR` parameter in the file `cdk.json`.
+
 
 ## DNS and Certificates
 
-Sage IT manages the creation of DNS records and TLS certificates in [org-formation](https://github.com/Sage-Bionetworks-IT/organizations-infra/tree/master/org-formation).
-The DNS records are managed centrally in the SageIT account, and corresponding
-wildcard TLS certificates are created in any application accounts that will
-deploy applications with custom (non-AWS) domains.
-
-When deploying a new application (or an existing application to a new account,
-e.g. the first deploy to "prod"), first check that a certificate for the
-desired domain has been created in the account the application will be deployed
-to (e.g. [here for app.sagebionetworks.org](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/100-shared-dns/_tasks.yaml#L24-L27)).
-If a certificate is needed in a new account, make a request to the IT team
-because there is a manual validation step that must be performed by an
-administrator. Set the value of `ACM_CERT_ARN` context variable to the ARN of
-the certificate in the target account. If the AWS ARN for an existing
-certificate is not known, it can be requested from Sage IT.
+Obtain the ARN of the ACM Certificate by following the instructions
+[here](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2859302913/Admin+Tasks+for+CDK+Applications)
```

Comment:
i think it would be better to generalize the wiki doc and move it to the organizations-infra repo here https://github.com/Sage-Bionetworks-IT/organizations-infra/tree/master/org-formation/800-redirects  then reference the doc in the github repo from here.

Reply:
The wiki doc' addresses more than just the `800-redirects/` part of our infra'. It summarizes a list of steps (both infra' changes and manual steps).  It's not obvious to me how to reorganize the content into GitHub.

Reply:
docs for manual steps can live in github as well.  i'm suggesting that we move the entire wiki doc (automation+manual steps) to github and make github the source of truth for that doc.